pyproject.toml

```
@@ -1,23 +1,182 @@
 [build-system]
-requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel"]
+requires = ["setuptools>=65.4.1", "setuptools_scm[toml]>=7", "wheel"]
```

```suggestion
requires = ["setuptools>=65.4.1", "setuptools_scm[toml]>=7"]
```
I think wheel isn't required here.